samza-sql/src/main/java/org/apache/samza/sql/data/Expression.java

```diff
@@ -29,10 +30,10 @@
 public interface Expression {
   /**
    * This method is used to implement the expressions that takes in columns as input and returns multiple values.
-   * @param context the context
+   * @param context SamzaSqlExecution context
```

Comment:
nit: the variable name should be 'sqlContext'
and need to add another param on the next line for regular 'context'